README.md

```diff
@@ -48,298 +48,400 @@ The FWS Payments contract enables ERC20 token payment flows through "rails" - au
-- With `lockupFixed = 3`, total required: `32 + 3 = 35 tokens`.
-- The client must top up the account by 8 tokens before this change is allowed.
+The operator needs to increase the payment rate to 4 tokens/epoch.
+- **Action:** `modifyRailPayment` is called with `newRate = 4`.
+- **New Lockup Requirement:** The rail's streaming lockup becomes `4 × 8 = 32 tokens`. The total requirement is now `32 + 3 = 35 tokens`.
+- **Funding Check:** This change increases the rail's lockup requirement by 8 tokens (from 27 to 35). The transaction will only succeed if the client's account has at least 8 tokens in available (non-locked) funds to cover this increase. If not, the call will revert.
 
 **Scenario 3: Reducing the Lockup Period**
-- The client reduces the lockup period to 5 epochs.
-- Streaming lockup: `3 × 5 = 15 tokens`.
-- With `lockupFixed = 3`, total required: `15 + 3 = 18 tokens`.
-- The contract now allows the client to withdraw any excess above 18 tokens.
+The operator reduces the lockup period to 5 epochs.
+- **Action:** `modifyRailLockup` is called with `period = 5`.
+- **New Lockup Requirement:** The streaming lockup becomes `3 × 5 = 15 tokens`. The total requirement is now `15 + 3 = 18 tokens`.
+- **Result:** The rail's total lockup requirement is reduced from 27 to 18 tokens. This frees up 9 tokens in the client's `Account.lockupCurrent`, which they can now withdraw (assuming no other lockups).
 
-#### Summary Table
 
-| Lockup Type      | What it Covers                | Example Use Cases                                     |
-|------------------|-------------------------------|-------------------------------------------------------|
-| Streaming        | Future periodic payments      | Subscriptions, ongoing services                       |
-| Fixed            | Immediate one-time payments   | Onboarding fees, bonuses, penalties, termination fees |
+#### Best Practices for Service Providers
 
-#### Best Practices
+This lockup mechanism places clear responsibilities on the service provider to manage risk:
 
-- **Plan ahead:** Estimate both your ongoing (streaming) and one-time payment needs and fund both buckets accordingly.
-- **Monitor balances:** Before making a one-time payment or increasing the streaming rate / period, check that your account is sufficiently funded.
-- **Use cases:** Use streaming lockup for predictable, recurring payments. Use fixed lockup for immediate payments.
-
-This model gives flexibility: this can guarantee ongoing service payments while also handling special, immediate payments as needed—all with clear, enforced separation and safety for both parties.
+-   **Settle Regularly:** You must settle rails frequently. A rail's `lockupPeriod` is a measure of the risk you are willing to take. If you wait longer than the `lockupPeriod` to settle, you allow a client to build up a payment obligation that may not be fully covered by the lockup guarantee if they become insolvent.
```

Comment:
"depending on the solvency guarantees put in place by the operator contract's lockup requirements"?

Reply:
Done.